python/lsst/ts/rubintv/handlers/ddv_websocket_handler.py

```diff
@@ -125,15 +127,16 @@ async def connect(self, websocket: WebSocket, client_type: str) -> str:
         websocket : `WebSocket`
             The WebSocket connection to register.
         client_type : `str`
```

Comment:
This param name should changed to `connection_type`.

Reply:
Thanks!